### PR TITLE
Bump PHP dependency versions

### PR DESCRIPTION
**Pull request by paddatrapper**
Bumps our PHP dependencies to new versions of upstream. Required to fix Zend failing on Ubuntu Focal (https://github.com/LibreTime/libretime/pull/1168#issuecomment-917685593). Also pinned the dependencies on git master branches to versions where recent releases have been made.

### CODE REVIEW

**Comment by jooola** [comment]
Possibly duplicate of #1354

**Comment by paddatrapper** [comment]
Some, but I'm not addressing PHP 7.4, or thinning out unused dependencies. Just trying to get new upstream versions in. I do not intend on touching the propel dependency here

File: airtime_mvc/composer.json [hunk 1]
```diff
@@ -15,36 +15,36 @@
     "propel/propel1": "1.7.0-stable",
     "aws/aws-sdk-php": "2.7.9",
     "raven/raven": "0.12.0",
-    "massivescale/celery-php": "2.0.*@dev",
-    "simplepie/simplepie": "dev-master",
-    "zf1s/zend-application": "^1.12",
+    "massivescale/celery-php": "^2.1",
+    "simplepie/simplepie": "1.5.6",
+    "zf1s/zend-application": "^1.13",
     "composer/semver": "^1.4",
```
Comment (jooola) [inline airtime_mvc/composer.json:21]: I was able to bump semver to the latest version, as none of the breaking changes affect us.

File: packages.ini [hunk 2]
```diff
@@ -23,6 +23,7 @@ php7.2 = bionic
 php7.2-curl = bionic
 php7.2-gd = bionic
 php7.2-pgsql = bionic
+php7.2-xml = bionic
```
Comment (jooola) [inline packages.ini:26]: Where does these come from ?

Comment (paddatrapper) [reply packages.ini:26]: simplepie requires it, but has only recently added a check that fails if it doesn't exist - https://github.com/simplepie/simplepie/pull/590

File: airtime_mvc/composer.json (hunk 1 above)
Comment (paddatrapper) [reply airtime_mvc/composer.json:21]: Bumped. If there are any others, we can probably include them in the PHP7.4 PR

**Pull request merged**